backend/api/Models/Concepts/AcquisitionFile/AcquisitionFilePersonModel.cs

```diff
@@ -10,15 +10,27 @@ public class AcquisitionFilePersonModel : BaseModel
         public long Id { get; set; }
 
         /// <summary>
-        /// get/set - The Id  of the person associated with acquisition file.
+        /// TODO: Remove/refactor this flattened model
+        /// get/set - The Id of the person associated with an acquisition file as part of the acquisition team.
```

Comment:
Same comment re: flattened models

Reply:
assuming that we come to an agreement, you can go ahead and remove these todos.